crates/bevy_utils/Cargo.toml

```
@@ -35,6 +35,9 @@ bevy_platform = { path = "../bevy_platform", version = "0.16.0-dev", default-fea
 
 thread_local = { version = "1.0", optional = true }
 
+[target.'cfg(all(target_arch = "wasm32", target_feature = "atomics"))'.dependencies]
+send_wrapper = "0.6.0"
```

`send_wrapper` is _not_ `no_std` compatible, so just add `"dep:send_wrapper"` to the `std` feature along with this change (and then feature-gate the wrapper with `#[cfg(feature = "std")]`

```suggestion
send_wrapper = { version = "0.6.0", optional = true }
```